api/client/service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message SendCommandRequest {
+  enum CommandType {
+    kStart = 0;
+    kUpdate = 1;
+    UNSET = 2;
+  }
+  CommandType command_type = 1;
+  CommandLineOptions options = 2;
+}
+
+message SendCommandResponse {
+  repeated Output output = 1;
```

Comment:
How come this is repeated?

Reply:
Whoops, stale artifact. Cleaned it up, this no longer has `repeated`.